pool/src/certora/model.rs

```diff
@@ -0,0 +1,19 @@
+static mut CHECKED: bool = false;
+static mut SKOLEM_I: u32 = 0;
+
+pub fn get_checked() -> bool {
+    unsafe { CHECKED }
+}
+pub fn set_checked() {
+    unsafe { CHECKED = true }
+}
+
+pub fn skolem_i() -> u32 {
+    unsafe { SKOLEM_I }
+}
+pub fn init() {
+    unsafe {
+        SKOLEM_I = nondet::nondet();
```

Comment:
should we add comments to this file to indicate what these are for? Also, should we prepend them with GHOST?